### DISCUSSION TITLE
Livewire file upload failing from mobile camera

### DISCUSSION THREAD

**Discussion by morganchorlton3** [livewire/livewire#1759] [4. General]
Hi, I am using laravel in an application running on mobile devices and I want to be able to upload images from the user's camera it fails most of the time it has worked like 1 in a 100 times. I'm really not sure what is going on is there a way to get more information on the error? i just get a validation error message files failed to upload

**Comment by ppegu** [comment]
what's your code ?

**Comment by thetwopct** [comment]
@morganchorlton3 Did you get any solution for this? I'm running in to a similar problem where I have a wire:model photo, and then using updatedPhoto to trigger the photo upload but it doesn't work. Using a saved photo or using the same upload on desktop works fine, just never for a "just taken" photo.

**Reply by morganchorlton3** [reply]
urm short answer no , longer answer is i tried a bunch of different thing and i think it came down to a combination of thing like updating livewire to the latest version, I also remember setting the camera to the default option on mobile devices rather than the allowing them to use the file browser. 

`   <input type="file" accept="image/*" capture="camera">`

luckily that was acceptable for the project I was working on. sorry this isnt much help it was on a project from a long time ago

**Comment by satoved** [comment]
Ok, it's always about the file size. Things to check if you encounter in again:
1) File uploading can return "413 Request Entity Too Large" so check you Nginx/php settings for that
2) Check livewire.temporary_file_upload.rules config, which is 12MB by default and change if needed
3) If you use spatie/laravel-media-library, also check media-library.max_file_size config

Remember, that Livewire sends base64 encoded file contents, which is much larger than the original file.
